File: examples/python/post__predict_time-series.py
# -*- coding: utf-8 -*-
import requests

# 示例：POST /predict/time-series
# 使用 POST 方法请求 /predict/time-series
url = "http://localhost:8000/api/v1/predict/time-series"
headers = {"Authorization": "Bearer YOUR_TOKEN"}

try:
    response = requests.post(url, headers=headers)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)

File: examples/js/post__predict_time-series.js

const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：POST /predict/time-series
fetch(`http://localhost:8000/api/v1/predict/time-series`, {
    method: 'POST',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));
